src/main/java/org/folio/innreach/batch/contribution/service/OngoingContributionEventProcessor.java

```diff
@@ -15,8 +15,9 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.scheduling.annotation.Async;
 import org.springframework.stereotype.Service;
-import org.folio.innreach.dto.Item;
 import org.folio.innreach.dto.Holding;
+import org.folio.innreach.dto.Instance;
+import org.folio.innreach.dto.Item;
```

Comment:
These 3 should probably be grouped with the `org.folio.innreach` items above in alpha order.

Reply:
changed